Source/DiabloUI/selgame.cpp

```diff
@@ -35,7 +35,7 @@ extern int provider;
 
 namespace {
 
-char title[32];
+const char *title = "";

```

Comment:
Can't we use `string` instead?

Reply:
That would just make extra copies. If you really want to use some something from the stl, `string_view` is the closest to what we're doing.

Reply:
Can we use `string_view` then?

Reply:
Eventually all of it should be moved to `string`/`string_view`, it's already what is being used internally in the translation handling as well as the `fmt` string building. Most of the warnings we get in MSVC is from the use of `char*` for string handling. I guess the next step in this process would be to adapt the string rendering code to expect `std::string`, and then take out all the in between conversions as a final stage.